automated install/basic-install.sh

```diff
@@ -376,37 +376,19 @@ package_manager_detect() {
 }
 
 select_rpm_php(){
+    local unsupported_dialog=0
```

Comment:
If we want to switch to POSIX one day, try to avoid `local` already now.
```suggestion
    unsupported_rpm_distribution_dialog=0
```